syntax = "proto3";

package logs;

message LogRequest {
  string service_name = 1;
}

message LogMessage {
  string content = 1;
  int64 timestamp = 2;
}

service LogService {
  rpc StreamLogs (LogRequest) returns (stream LogMessage); // Server Streaming RPC
}
